### DISCUSSION TITLE
Grouped Stacked Bar Chart - border radius applied only to last element of series

### DISCUSSION THREAD

**Discussion by adilvalizada** [apexcharts/apexcharts.js#4649] [Q&A]
I have an issue about border at Grouped Stacked bar chart. I can't add border radius at all bar column. 

<img width="645" alt="Screenshot 2024-08-20 at 14 30 11" src="https://github.com/user-attachments/assets/9c530c9a-124f-4dc3-97d9-db569fa39608">

<img width="592" alt="Screenshot 2024-08-20 at 16 08 03" src="https://github.com/user-attachments/assets/b896456a-0573-40e9-8bda-09f83976cd8e">

**Comment by Mahdisaghroun** [comment]
@adilvalizada  Could you please share your code so I can take a closer look and help you out?

**Reply by adilvalizada** [reply]
`export const stackedComboConfigOptions = ({
  dynamicSeries,
}: ChartOptionsType) => {
  return {
    chart: {
      id: 'basic-bar',
      stacked: true,
      toolbar: {
        show: false,
      },
    },
    stroke: {
      width: dynamicSeries?.map((series) =>
        (series as SeriesItem)?.type === 'line' ? 5 : 1,
      ),
    },
    plotOptions: {
      bar: {
        columnWidth: '90%', // Adjust this value to control the width of bars
        horizontal: false,
        borderRadius: 10,
        borderRadiusApplication: 'end', // "around" / "end"
        borderRadiusWhenStacked: 'last', // "all"/"last"
        colors: {
          backgroundBarOpacity: 1,
        },
      },
    },
    yaxis: {
      min: stackedComboMinMax(dynamicSeries).minValue,
      max: stackedComboMinMax(dynamicSeries).maxValue,
      tickAmount: 5,
      labels: {
        maxWidth: 160,
        align: 'left',
        formatter(value: number) {
          return convertToKAndM(value);
        },
        style: {
          padding: '10px',
          fontFamily: styles.fontFamily,
          fontSize: '14px',
          fontWeight: '400',
        },
      },
    },
  };
};`

**Reply by adilvalizada** [reply]
Thanks for your reply, I would be grateful if you could help me.

**Comment by web-cloudy** [comment]
Hi, @adilvalizada 

As an expert, I can say no problem. Could you share your code?
I will take only 30 mins.

Good luck :)

**Reply by adilvalizada** [reply]
`export const stackedComboConfigOptions = ({
  dynamicSeries,
}: ChartOptionsType) => {
  return {
    chart: {
      id: 'basic-bar',
      stacked: true,
      toolbar: {
        show: false,
      },
    },
    stroke: {
      width: dynamicSeries?.map((series) =>
        (series as SeriesItem)?.type === 'line' ? 5 : 1,
      ),
    },
    plotOptions: {
      bar: {
        columnWidth: '90%', // Adjust this value to control the width of bars
        horizontal: false,
        borderRadius: 10,
        borderRadiusApplication: 'end', // "around" / "end"
        borderRadiusWhenStacked: 'last', // "all"/"last"
        colors: {
          backgroundBarOpacity: 1,
        },
      },
    },
    yaxis: {
      min: stackedComboMinMax(dynamicSeries).minValue,
      max: stackedComboMinMax(dynamicSeries).maxValue,
      tickAmount: 5,
      labels: {
        maxWidth: 160,
        align: 'left',
        formatter(value: number) {
          return convertToKAndM(value);
        },
        style: {
          padding: '10px',
          fontFamily: styles.fontFamily,
          fontSize: '14px',
          fontWeight: '400',
        },
      },
    },
  };
};`

**Reply by adilvalizada** [reply]
Hi Thanks for response , i am using this graphs (https://apexcharts.com/react-chart-demos/column-charts/grouped-stacked-columns/) i need solution for this. do you have an idea? how can i add border on this graph?

**Comment by Awais95-Dev** [comment]
Hi, @adilvalizada
I have solution to this. See the attachment, Is that what you want?
![a6](https://github.com/user-attachments/assets/d5f86d1d-b970-41c7-8372-363af70983d8)

**Reply by adilvalizada** [reply]
Hi , Thanks for response, i need solution for grouped stacked bar (appexchartjs) https://apexcharts.com/react-chart-demos/column-charts/grouped-stacked-columns/
